lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -145,4 +145,29 @@ def table_alias_length
     63
   end
   alias index_name_length table_alias_length
+
+
+  private
+
+    # Extracts the value from a PostgreSQL column default definition.
+    # Some of postgresql's default regexps are not going to work with cockroachdb,
+    # leaving the default values out of the schema.rb file.
+    def extract_value_from_default(oid, default) # :nodoc:
+      ret = super
+      if ret.nil?
+        puts "CIAO #{default} of #{oid}"
+        ret = case default
+          # Numeric types are identified in a different way by cockroachdb
+          # postgresql adapter only looks for ::bigint, while cockroach
+          # has INT and DECIMAL.
+          when /\A\(?(-?\d+(\.\d*)?)\)?(:::(INT(4,8,16,32,64)?|DECIMAL))?\z/
```

Comment:
Thank you! 👏 

We also allow the use of `1::INT` (2 colons is a type cast, 3 is a type annotation). Could you change the last colon to `:?` condition for now.